### PR TITLE
Pull request from Angela Duran morse translator

### PR DESCRIPTION
**Pull request by duranangela**

### CODE REVIEW

File: lib/translator.rb [hunk 1]
```diff
@@ -39,4 +41,23 @@ def initialize
                     "0" => "-----",
                     " " => " "}
   end
-end
\ No newline at end of file
+
+  def eng_to_morse(english)
+    english = english.downcase
+    array = []
```
Comment (BrianZanti) [inline lib/translator.rb]: This should not just be named array

File: lib/translator.rb [hunk 2]
```diff
@@ -39,4 +41,23 @@ def initialize
...
+    english.chars.each do |char|
+      new_char = @dictionary[char]
```
Comment (BrianZanti) [inline lib/translator.rb]: This should not just be named new_char. Name it something like morse_char

File: lib/translator.rb [hunk 3]
```diff
@@ -39,4 +41,23 @@ def initialize
...
+      array << new_char
+    end
+    array.join
+  end
+
+  def from_file(filename)
+    data = File.read("./lib/#{filename}").chomp
```
Comment (BrianZanti) [inline lib/translator.rb]: You should just take the filename as is. Don't assume it's in ./lib

**Comment by ameseee** [comment]
Looks good @duranangela - a few comments for your review.

**Pull request closed without merging**